umm-spec-lib/resources/xml-schemas/iso19115_2/resources/Codelist/eosCodelists.xml

```diff
@@ -228,6 +228,18 @@
                <gml:identifier codeSpace="gov.nasa.esdis.umm">Other</gml:identifier>
             </gmx:CodeDefinition>
          </gmx:codeEntry>
+         <gmx:codeEntry>
+            <gmx:CodeDefinition gml:id="EOS_AssociationTypeCode_isPreviousVersionOf">
+               <gml:description>This collection has been superseded by the Associated DOI.</gml:description>
+               <gml:identifier codeSpace="gov.nasa.esdis.umm">Is_Previous_Version_Of</gml:identifier>
+            </gmx:CodeDefinition>
+         </gmx:codeEntry>
+         <gmx:codeEntry>
+            <gmx:CodeDefinition gml:id="EOS_AssociationTypeCode_isNewVersionOf">
+               <gml:description>The collection supersedes the Associated DOI.</gml:description>
+               <gml:identifier codeSpace="gov.nasa.esdis.umm">Is_New_Version_Of</gml:identifier>
+            </gmx:CodeDefinition>
+         </gmx:codeEntry>
```

Comment:
Great Job!  We already had our own code list for this.

Reply:
What do you mean?

Reply:
eosCodelists.xml are our lists unique to our project. They differ from the standards, but that is OK, as that is what they are for.